Sources/SwiftParser/TokenPrecedence.swift

```diff
@@ -159,7 +159,7 @@ public enum TokenPrecedence: Comparable {
       // Secondary parts of control-flow constructs
       .keyword(.case), .keyword(.catch), .keyword(.default), .keyword(.else),
       // Return-like statements
-      .keyword(.break), .keyword(.continue), .keyword(.fallthrough), .keyword(.return), .keyword(.throw), .keyword(.yield),
+      .keyword(.break), .keyword(.continue), .keyword(.fallthrough), .keyword(.return), .keyword(.throw), .keyword(.yield), .keyword(.try),
```

Comment:
Maybe this is iffy, but without it it was giving `CanBeStatementStart`'s recovery precedence that of an expression as calculated [here](https://github.com/apple/swift-syntax/blob/b931534a93a21e1ceda00495ab6cb5ff4d8bd649/Sources/SwiftParser/Recovery.swift#L104). An alternative could to give `RawTokenKindSubset` a customization point and have the one for `CanBeStatementStart` exclude `try`.

Reply:
Raising the precedence of `try` seems totally correct to me. If `try` can start a statement now, we shouldn’t be eating it while looking for a statement keyword.